Tests/Reader/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+cmake_minimum_required(VERSION 3.10)
+project (reader)
+include_directories(/home/mari/Work/Phoebus/phoebus/Tests/Reader/)
+add_executable (output /home/mari/Work/Phoebus/phoebus/Tests/Reader/reader.cpp /home/mari/Work/Phoebus/phoebus/Tests/Reader/main.cpp /home/mari/Work/Phoebus/phoebus/Tests/Reader/reader.hpp) 
```

Comment:
This look like absolute paths. As a general rule, use relative paths when you can. Otherwise this file will only work on your computer and won't be portable.

Reply:
I assume that with the reader integrated in phoebus, this test can go away.

Alternatively, we can add the test to the phoebus unit tests. I can explain how to do that.